webapp/src/pages/Settings.tsx

```diff
@@ -165,6 +177,56 @@ const displayReadonlyFields = (label: string, value: string | null) => (
     }}
   />
 );
+const SelectField: React.FC<
+  Omit<SelectProps, "onChange"> & {
+    label: string;
+    value: any;
+    onChange: (newValue: any) => void;
+    noneValue?: string;
+    children: React.ReactElement<typeof MenuItem>[];
+  }
+> = ({ label, value, onChange, children, noneValue, ...props }) => (
+  <FormControl variant="standard" className="fixedWidthInput">
```

Comment:
The `model_contract` and `spacy_model` values don't fit in the `fixedWidthInput`. Maybe just pass `className="fixedWidthInput"` to the language `SelectField`?
```suggestion
  <FormControl variant="standard">
```

Reply:
I resolved this in #494

Reply:
Thank you :)